shell-config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   modules: ['botfuel-module-facetedsearch'],
   path: __dirname,
+  logger:'debug'
```

Comment:
are you sure?

Reply:
should remove this, we added it for debugging purpose only

Reply:
Was for debugging purposes, will delete.